routes/toppings.js

```diff
@@ -0,0 +1,14 @@
+import models from '../lib/models'
+import express from 'express'
+
+const router = express.Router()
+
+router.get('/', (req, res) => {
+  models.Toppings.findAll()
```

Comment:
This is fine, but you can also do this with (IMO cleaner) `async/await` syntax, it would look like this:
```
router.get('/', async (req, res) => {
  let toppings
  try {
    toppings= await models.Toppings.findAll()
  } catch (err) {
    console.log(err)
    res.sendStatus(500) // or something else in the 5XX series
  }
  if (toppings) {
    res.send(toppings)
  } else {
    res.sendStatus(404) // some not found error status, this could also be a 5XX
  }
})
```
I like the above since it's more "traditional" syntax, but the way that you have it here is shorter. If you choose to leave it the way you have it, that's fine, but you do need to add a `.error()` to the chain, eventually Node will crash when rejected promises are not caught.